.pre-commit-config.yaml

```diff
@@ -28,6 +26,7 @@ repos:
       - id: check-vcs-permalinks
       - id: check-xml
       - id: check-yaml
+        args: ["--unsafe"]
```

Comment:
Why do we need the unsafe?

Reply:
Otherwise, the checks, fail. I have no idea what the error message even means :sweat_smile: 
![image](https://user-images.githubusercontent.com/49768113/184925219-b1853c81-3d51-4f01-baee-970b9e3810b8.png)

Reply:
k ¯\\\_(ツ)_/¯